app/assets/stylesheets/application.scss

```diff
@@ -1,5 +1,7 @@
 @import "govuk_publishing_components/all_components";
 
+@include _govuk-font-face-nta;
```

Comment:
It would be helpful to see an example of something that was broken that this fixes. Also, I'm wondering how anyone else would know they needed to add this in another app - that might just be my lack of experience with govuk-frontend, though!

Reply:
The example is https://content-publisher.integration.publishing.service.gov.uk/ using the fallback font. It probably makes more sense for me to extract this into a separate PR.

Reply:
👍

Reply:
Moved it to https://github.com/alphagov/content-publisher/pull/917